rotkehlchen/api/v1/encoding.py

```diff
@@ -1274,6 +1275,7 @@ class UserPremiumSyncSchema(AsyncQueryArgumentSchema):
 class NewUserSchema(BaseUserSchema):
     premium_api_key = fields.String(load_default='')
     premium_api_secret = fields.String(load_default='')
+    sync_database = fields.Boolean(load_default=True)
```

Comment:
Should the default at user creation be to sync the database from premium? I think yes. @kelsos what do you think?